release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1460,6 +1464,64 @@ metadata:
++
+[source,terminal]
+----
+[ERROR]: Detected a v2 `ImageSetConfiguration`, please use `--v2` instead of `-v2`.
+----
++
+(link:https://issues.redhat.com/browse/OCPBUGS-33121[*OCPBUGS-33121*])
+
+* Previously, oc-mirror plugin v2 did not automatically perform retries when it encountered errors on registries, such as timeouts, expired authentication tokens, HTTP 500 errors, and so on. With this update, retries for these errors are implemented, and users can configure retry behavior with the following flags:
++
+* `--retry-times`: Specifies the number of retry attempts. Default is 2.
+* `--retry-delay`: Sets the delay between retries. Default is 1 second.
+* `--image-timeout`: Defines the timeout period for mirroring an image. Default is 10 minutes.
+* `--max-parallel-downloads`: Controls the maximum number of layers to pull simultaneously during a single copy operation. Default is 6.
++
+(link:https://issues.redhat.com/browse/OCPBUGS-34021[*OCPBUGS-34021*])
+
+* Previously, when using the oc-mirror plugin v2 with the `--rebuild-catalogs` flag, the catalog cache was regenerated locally, which caused failures either due to compatibility issues with the `opm` binary and the platform or due to cache integrity problems on the cluster. With this update, the `--rebuild-catalogs` flag defaults to true, so catalogs are rebuilt without regenerating the internal cache. Additionally, the image command has been modified to generate the cache during pod startup, which may delay pod initialization. (link:https://issues.redhat.com/browse/OCPBUGS-37667[*OCPBUGS-37667*])
+
+* Previously, the oc-mirror plugin v2 did not use the system proxy configuration to recover signatures for releases when running behind a proxy with system proxy settings. With this release, the system proxy settings are now applied during the signature recovery process. (link:https://issues.redhat.com/browse/OCPBUGS-37055[*OCPBUGS-37055*])
+
+* Previously, oc-mirror plugin v2 would stop the mirroring process when it encountered operators using bundle versions that were not compliant with semantic versioning, which also prevented the creation of cluster resources like IDMS, ITMS, and `CatalogSource` objects. With this fix, the plugin now skips these problematic images instead of halting the process. If an image uses incorrect semantic versioning, a warning message is displayed in the console with the relevant image details. (link:https://issues.redhat.com/browse/OCPBUGS-33081[*OCPBUGS-33081*])
```

Comment:
```suggestion
* Previously, oc-mirror plugin v2 would stop the mirroring process when it encountered Operators using bundle versions that were not compliant with semantic versioning, which also prevented the creation of cluster resources like IDMS, ITMS, and `CatalogSource` objects. With this fix, the plugin now skips these problematic images instead of halting the process. If an image uses incorrect semantic versioning, a warning message is displayed in the console with the relevant image details. (link:https://issues.redhat.com/browse/OCPBUGS-33081[*OCPBUGS-33081*])
```